internal/commands/untrust_builder.go

```diff
@@ -17,6 +17,12 @@ func UntrustBuilder(logger logging.Logger, cfg config.Config) *cobra.Command {
 		Args:  cobra.ExactArgs(1),
 		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
 			builderName := args[0]
+
+			if !isTrustedBuilder(cfg, builderName) {
```

Comment:
This command iterates through `cfg.TrustedBuilders` which we do again on line 28. Given discussions around the inefficiency of this data structure being a list we need to iterate through, is it possible to at least only do that once?

Reply:
I moved `isTrustedBuilder` to within the no-trusted-builder case, and pass in an empty config to maximize efficiency.

Reply:
Please see my idea in [this comment](https://github.com/buildpacks/pack/pull/725#discussion_r449322973).